pymove/utils/integration.py

```diff
@@ -0,0 +1,1008 @@
+    for Points of Interest in a single category named 'bus_station'.
+
+    Parameters
+    ----------
+    df_ : dataframe
+        Input points of interest data
+
+    label_poi : String, optional("type_poi" by default)
+        Label referring to the Point of Interest category
+
+    Returns
+    -------
+    """
+
+    print('union bus station categories to one category')
+    filter_bus_station = df_[label_poi].isin(
+        ['transit_station', 'pontos_de_onibus']
+    )
+    df_.at[df_[filter_bus_station].index, label_poi] = 'bus_station'
+
+
+def union_poi_bar_restaurant(df_, label_poi='type_poi'):
+
+    """
+    Performs the union between bar and restaurant categories
+    for Points of Interest in a single category named 'bar-restaurant'.
+
+    Parameters
+    ----------
+    df_ : dataframe
+        Input points of interest data
+
+    label_poi : String, optional("type_poi" by default)
+        Label referring to the Point of Interest category
+
+    Returns
+    -------
+    """
+
+    print('union restaurant and bar categories to one category')
+    filter_bar_restaurant = df_[label_poi].isin(['restaurant', 'bar'])
+    df_.at[df_[filter_bar_restaurant].index, label_poi] = 'bar-restaurant'
+
+
+def union_poi_parks(df_, label_poi='type_poi'):
+
+    """
+    Performs the union between park categories
+    for Points of Interest in a single category named 'parks'.
+
+    Parameters
+    ----------
+    df_ : dataframe
+        Input points of interest data
+
+    label_poi : String, optional("type_poi" by default)
+        Label referring to the Point of Interest category
+
+    Returns
+    -------
+    """
+
+    print('union parks categories to one category')
+    filter_parks = df_[label_poi].isin(['pracas_e_parques', 'park'])
+    df_.at[df_[filter_parks].index, label_poi] = 'parks'
+
+
+def union_poi_police(df_, label_poi='type_poi'):
+
+    """
+    Performs the union between police categories
+    for Points of Interest in a single category named 'police'.
+
+    Parameters
+    ----------
+    df_ : dataframe
+        Input points of interest data
+
+    label_poi : String, optional("type_poi" by default)
+        Label referring to the Point of Interest category
+
+    Returns
+    -------
+    """
+
+    print('union distritos policies and police categories')
+    filter_police = df_[label_poi] == 'distritos_policiais'
+    df_.at[df_[filter_police].index, label_poi] = 'police'
+
+<<<<<<< HEAD
+def join_coletive_areas(gdf_, gdf_rules_, label_geometry='geometry'):
+    
+    """
+    It performs the integration between trajectories and collective
+    areas, generating a new column that informs if the point of the
+    trajectory is inserted in a collective area.
+    
+=======
+
+def join_coletives_areas(gdf_, gdf_rules_, label_geometry='geometry'):
+
+    """
+    Performs the integration between trajectories and coletives areas.
+
+>>>>>>> f032a4301129a29fa2bc7132092b75d9fcd8ed5e
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible SCM diff in code (diff)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5610065)